storage/mock/src/main/java/com/emc/mongoose/storage/mock/impl/http/request/RequestHandlerBase.java

```diff
@@ -109,7 +118,8 @@ protected RequestHandlerBase(
 		this.localStorage = localStorage;
 		this.contentSource = contentSource;
 		this.ioStats = localStorage.getStats();
-		apiClsName = getClass().getSimpleName();
+		this.apiClsName = getClass().getSimpleName();
+		storagePoller = Executors.newCachedThreadPool();
```

Comment:
why not ThreadPoolExecutor instance?
if there's no remote nodes count known we have to have an ability to change the thread pool size on the fly